website/docs/concepts/canbus_service/README.md

```diff
@@ -5,7 +5,7 @@ title: Canbus
 
 # Canbus Service Overview
 
-The Canbus service is an integral component of the Amiga platform.
+The Canbus sub-service is an integral component of the Amiga platform.
```

Comment:
It's not a sub-service yet, but I'm fine with keeping this if you don't want to have to remember to change it later

Reply:
This was an oversight. Reverted here: ca537559f3e029df6f8e715dfa405924d6f4285f